Nightscouter/SiteDetailViewController.swift

```diff
@@ -141,35 +141,30 @@ extension SiteDetailViewController {
     
     func updateSite(_ notification: Notification?) {
         print(">>> Entering \(#function) <<<")
-        self.updateData(forceUpdate: true)
+        self.updateData()
     }
     
-    func updateData(forceUpdate force: Bool = false) {
+    func updateData() {
         guard let site = self.site else { return }
+        UIApplication.shared.isNetworkActivityIndicatorVisible = true
         
-        if (site.updateNow || site.sgvs.isEmpty || force == true) {
```

Comment:
It seems like, at least in the scope of this change, we're only ever passing in `true` to `updateData(Bool)`. We only call the method from `updateSite()` and `true` is hard coded.